File: FDP/T5_A5_PinedoBerumenClaraRuby/Ejercicio06_Figura.py
'''
Created on 4 dic. 2019

@author: Ruby
'''
print("Programa que muestra una figura con la altura ingresada por el usuario\n")

altura = int (input("Ingresa la altura:"))

for i in range(altura):
    for j in range(10):
        print("@",end=" ")
    print("\n")

File: FDP/T3_A6_PinedoBerumenClaraRuby/src/Ejercicio06_Figura.java

public class Ejercicio06_Figura {

	public static void main(String[] args) {
		
		
		System.out.println("==Programa que muestra un triangulo==");
		System.out.println();
		
		for(int i=20; i>0; i--) {
			for(int x=0; x<i; x++) {
				System.out.print("*");
			}
			
			System.out.println();
			
			for(int y=20-i; y>=0; y--) {
				System.out.print(" ");
			}
			
			
		}


	}

}
